index.js

```diff
@@ -137,10 +175,15 @@ function downloadAndPipeIntoStdout(url){
 }
 
 function getArrayOfEpisodes(source, input){
-    const rawEpisodes = input.split(',').map(x => parseRange(x.trim())).reduce((acc,val) => acc.concat(...val), [])
-    const uniqueEpisodes = uniqueArray(rawEpisodes.sort()).map(x => `Episode ${x}`)
-
-    return uniqueEpisodes.map(x => source[x])
+    let episodeArray;
+    if(input != "*"){
+        const rawEpisodes = input.split(',').map(x => parseRange(x.trim())).reduce((acc,val) => acc.concat(...val), [])
+        const uniqueEpisodes = uniqueArray(rawEpisodes.sort()).map(x => `Episode ${x}`)
+        episodeArray = uniqueEpisodes.map(x => source[x]);
+    }else{
+        episodeArray = Object.values(source);
+    }
+    return episodeArray;
```

Comment:
These are just minor code-style changes

```suggestion
    if(input != '*'){
        const rawEpisodes = input.split(',').map(x => parseRange(x.trim())).reduce((acc,val) => acc.concat(...val), [])
        const uniqueEpisodes = uniqueArray(rawEpisodes.sort()).map(x => `Episode ${x}`)
        return uniqueEpisodes.map(x => source[x]);
    }else{
       return Object.values(source)
    }
```